label.go

```diff
@@ -15,6 +15,9 @@ type Label struct {
 
 	text     string
 	wordWrap bool
+
+	// cache the result of SizeHint() (see #14)
```

Comment:
I fine either way I was just trying to signal that the field below is "just" used as a cache unlike the private field above. It is a *really* private field :-)